internal/testrunner/runners/system/runner.go

```diff
@@ -1333,6 +1376,18 @@ func (r *runner) validateTestScenario(ctx context.Context, result *testrunner.Re
 		return result.WithError(err)
 	}
 
+	// TODO: remove this once Elasticsearch will map event.original correctly (https://github.com/elastic/elasticsearch/pull/106714)
+	ignoredFields := make([]string, 0, len(scenario.ignoredFields))
+	for _, field := range scenario.ignoredFields {
+		if field != "event.original" {
+			ignoredFields = append(ignoredFields, field)
+		}
+	}
+
+	if len(ignoredFields) > 0 {
+		return result.WithError(fmt.Errorf("found ignored fields in data stream %s: %v", scenario.dataStream, ignoredFields))
+	}
```

Comment:
If looks like this change is going to be released in 8.14.0, and I guess we need to keep this code for older versions, so we can add an if to decide if we need to run this code, and remove the TODO:
```suggestion
	stackVersion, err := semver.NewVersion(serviceOptions.StackVersion)
	if err != nil {
		....
	}
	if stackVersion.LessThan(semver.MustParse("8.14.0")) {
		ignoredFields := make([]string, 0, len(scenario.ignoredFields))
		for _, field := range scenario.ignoredFields {
			if field != "event.original" {
				ignoredFields = append(ignoredFields, field)
			}
		}
		if len(ignoredFields) > 0 {
			return result.WithError(fmt.Errorf("found ignored fields in data stream %s: %v", scenario.dataStream, ignoredFields))
		}
	}
```